src/basho_bench_stats.erl

```diff
@@ -111,6 +113,8 @@ init([]) ->
     %% successful operations
     [begin
          folsom_metrics:new_histogram({latencies, Op}, slide, basho_bench_config:get(report_interval)),
+         folsom_metrics:new_histogram({overall_latencies, Op}, uniform, 16384),
```

Comment:
Originally I started with 1024, but I figured that if we collect p999, we should have some more datapoints than just ~1, so I figured approx 10x  and then round up to the next power of two.

Its a bit arbitrary, but that was my thought process